docs/dictionary/function/processor.lcdoc

```diff
@@ -23,17 +23,15 @@ if the processor is "ARM" then ...
 
 Returns (enum):
 
--   x86: Windows systems (any architecture) and Mac OS X and Linux
-    systems with a 32 bit processor.
--   x86_64: Mac and Linux systems running a 64 bit processor
--   i386: iOS Simulator.
--   ARM: iOS and Android Devices and the Android simulator
-
+-   x86: 32 or 64 bit Windows systems and 32 bit macOS or Linux systems.
```

Comment:
I think it probably should be the arch of the running binary and not the machine - as it determines how the application runs code. For example, on Windows, 32-bit and 64-bit things are entirely separate; just as they are no Linux. If you are running a 32-bit app in a 64-bit OS, then paths to things might be different (and will be correct if you are using the arch of the binary).